src/main/java/com/agilethought/atsceapi/model/UserModel.java

```diff
@@ -0,0 +1,46 @@
+package com.agilethought.atsceapi.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "USERS")
```

Comment:
Do we need to define a Table? Due to we are using NoSQL data base this annotation should be `@Document`